### PR TITLE
Add TITLE CASE.PY

### PR DESCRIPTION
**Pull request by Priyanshu729**
IN AN EASY MANNER

### CODE REVIEW

File: python/TITLE CASE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+def title_case(name):
+      size=len(name)
+      a=" "
+      for i in range(size):
+            if name[i]="" or name[i]==".":
```
Comment (tasnimzotder) [inline python/TITLE CASE]: hey @Priyanshu729 we're getting errors

- change `if name=""` to `if name==""`
- and correct the indentations which is `4 spaces`. also include a screenshot of the output

Comment (Priyanshu729) [reply python/TITLE CASE]: i will solve that error can i resend it??

**Review by Priyanshu729 — COMMENTED** [review]
- 

- 

1. 

- [ ] #71  solved

**Review by tasnimzotder — APPROVED** [review]
Thanks @Priyanshu729 for your interest in open source.

**Pull request merged**